src/Mount.cpp

```diff
@@ -3799,3 +3811,54 @@ void Mount::checkRALimit()
     }
     _lastTRKCheck = millis();
 }
+
+#if DEW_HEATER == 1
+/////////////////////////////////
+//
+// setHeater
+//
+/////////////////////////////////
+void Mount::setHeater(unsigned num, unsigned val)
+{
+    unsigned pin = 0, max = 0;
+    if (num == 0)
+    {
+        pin = DEW_HEATER_1_PIN;
```

Comment:
This should either check to make sure DEW_HEATER_N_PIN is defined, or ConfigurationValidation should be modified to make sure both pins are defined if DEW_HEATER is enabled.

Reply:
DEW_HEATER_N_PIN is always set due to the way it's set up the board files?

Reply:
Ideally yes, but I can easily see someone breaking it with a custom pins file or something. Just a precaution.